sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -88,4 +86,15 @@ public void delete(Long id) {
         }
         return dtoList;
     }
+
+    @Override
+    public TreeMap<String, String> getViewsByDiscounts() {
```

Comment:
`TreeMap` is too specific implementation. It is better to return simple `Map`, TreeMap, HashMap should be used only when we create instances

Reply:
Done. Changed to Map.